doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -205,18 +205,15 @@ In Keystone v1 and v2 (both deprecated), the term "tenant" was used in OpenStack
-Availability zones: provide resiliency and fault tolerance for VM deployments, by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms
+### 3.3.3. Cloud partitioning: Host Aggregates, Availability Zones
+Cloud administrators can partitioned the hosts within an OpenStack cloud using 2 mechanisms: Host Aggregates and Availability Zones.
 
-Host aggregate: is a Cloud Admin concept which is used to map the VNFC instances on the compute nodes
+A Host Aggregate is a group of hosts (compute nodes) with specific characteristics, software and/or hardware properties. Example would be a Host Aggregate created for specific hardware or performances. The administrator assign key-value pairs to groups of compute nodes, this information will be used to schedule VMs. A host can belong to multiple host aggregates. Host Aggregates are not explicitly exposed to tenants.
 
-A host aggregate is a set of hosts with specific properties (multiple software and/or hardware properties); the properties are specified as key-value pairs. Example would be a host aggregate created for a particular flavour or specific hardware. A host can belong to multiple host aggregates. Host aggregates are not visible to users.
+Availability Zones (AZs) are another mechanism for partitioning. They are exposed to tenants. They are defined by attaching specific metadata information to an aggregate, making the aggregate visible for tenants. Hosts can only be in a single Availability Zone. By default a host is part of a default Availability Zone, even if it doesn’t belong to an aggregate. Availability Zones are used to provide resiliency and fault tolerance for VM deployments, by means of physical hosting distribution of Compute Nodes in separate racks with separate power supply and eventually in different rooms. They permit rolling upgrades – an AZ at a time upgrade with enough time between AZ upgrades to allow recovery of tenant workloads on the upgraded AZ. Finally, AZs can accommodate hosts with special hardware and software characteristics, for example, hosts with hardware accelerators.
 
-Availability Zones are user visible host aggregates where a host can only be in one availability zone. Availability zones partition the cloud independent of the infrastructure layout. Availability zones (AZ) serve a couple of important purposes. Firstly, users can deploy their workloads to create local redundancy for resiliency and high availability. This permits rolling upgrades – an AZ at a time upgrade with enough time between AZ upgrades to allow recovery of tenant workloads on the upgraded AZ. Secondly, AZs can accommodate hosts with special hardware and software characteristics, for example, hosts with hardware accelerators.
+An over use of Host Aggregates and Availability Zones can result in a granular partition the cloud and, hence, operational complexities and inefficiencies.
```

Comment:
do you want to add some examples of "inefficiencies" such as in resource utilisation, operational complexities and inefficiencies?

Reply:
@pgoyal01 Yes, it will be good